index.bs

```diff
@@ -45,6 +45,16 @@ application runs a machine learning model such as [[DeepLabv3+]] or
 [[MaskR-CNN]] to semantically split an image into segments and replaces
 segments that represent other people and background with another picture.
 
+A live streaming application offers [[Bullet-Chatting]], a user-generated dynamic
+comments float over a video or static image feature, which may disturb users from
+watching the media content. The [[Masking-Bullet-Chatting]] could semantically
```

Comment:
s/The [[Masking-Bullet-Chatting]] could semantically split video into masks/The video could be split semantically into masks/